lib/engine/corporation.rb

```diff
@@ -255,6 +255,14 @@ def share_percent
       @second_share&.percent || presidents_percent / 2
     end
 
+    def border?
```

Comment:
why?

Reply:
I'm adding the :blocks_hexes ability to corporations (as opposed to just companies). They are now allowed to be blockers. Both Engine::Tile, UI::Blocker and UI::LocationName expect blockers to have a border? method.

Reply:
All right. I think I can avoid this if I can change Engine::Tile to not call the border? method for blockers. Then border? can be removed on both corporations and companies.

Reply:
Or not, I need to understand why the UI does this:
@tile.parts.reject(&:border?).one?

Reply:
Removed blockers from @parts